sdcm/remote.py

```diff
@@ -243,7 +254,26 @@ def _ssh_ping(self, timeout=30, verbose=False):
             self.log.debug(details)
             return False
 
-    def receive_files(self, src, dst, delete_dst=False,  # pylint: disable=too-many-arguments,too-many-branches
+    def ssh_ping_thread(self):
+        while not self._ssh_up_thread_termination.isSet():
+            result = self._ssh_ping()
+            if result:
+                self._ssh_is_up.set()
+            else:
+                self._ssh_is_up.clear()
+            time.sleep(5)
```

Comment:
please replace this `time.sleep` with `self._ssh_up_thread_termination.wait(5)` so if _ssh_up_thread_termination is set the loop stops immediately

Reply:
@aleksbykov ping

Reply:
Fixed